resources/tags.yaml

```diff
@@ -0,0 +1,171 @@
+accounting:
+  title: 'Accounting'
+  description: 'Processing and communication of financial information'
+analytics:
+  title: 'Analytics'
+  description: 'Data or statistics to make intelligent marketing and business decisions'
+backend: 
+  title: 'Backend'
+  description: 'Backend'
+blogging: 
+  title: 'Blogging'
+  description: 'Informational site published with discrete entries'  
+bookmarks:
+  title: 'Bookmarks'
+  description: 'Uniform Resource Identifier that is stored for later retrieval'
```

Comment:
covered by `social`, for example `pinterest`
something like `hacker news` can go under `media`